project/Build.scala

```diff
@@ -33,15 +33,14 @@ object MinimalBuild extends Build {
     resolvers += typesafeSnapshot,
     resolvers += sonatypeSnapshot,
     // publishArtifact in packageDoc := false,
-    crossScalaVersions := Seq("2.10.4","2.11.5"),
-    scalaVersion := "2.11.5",
+    scalaVersion := "2.11.7",
```

Comment:
No, there is no specific reason. 
I'd appreciate it if original project gets upgraded to akka-2.4. Can you please give a pull request?